### PR TITLE
mem-ruby, gpu-compute: fix formatting of TCC

### PR DESCRIPTION
**Pull request by mattsinc**
mem-ruby, gpu-compute: fix formatting of TCC

Fix several not properly indented prints and extraneous extra lines in the SLICC code for the GPU TCC (L2 cache).

Change-Id: I6ef84960e7054ae4f91ae50df701dec7c74de8db

### CODE REVIEW

**Comment by powerjg** [comment]
Looks fine, but it needs to be rebased.

**Comment by mattsinc** [comment]
> Looks fine, but it needs to be rebased.

done, I think

**Review by powerjg — DISMISSED** [review]
Looks good to me. Thanks!

**Comment by powerjg** [comment]
For whoever commits this, make sure to include the correct tags in the commit message.

**Comment by mattsinc** [comment]
> For whoever commits this, make sure to include the correct tags in the commit message.

Better?

**Comment by powerjg** [comment]
That's good. Thanks, Matt. I *think* this can be updated when the maintainer merges, which will reduce the amount of re-running the tests. By changing the commit message, the tests re-run, I think.

**Comment by mattsinc** [comment]
> That's good. Thanks, Matt. I _think_ this can be updated when the maintainer merges, which will reduce the amount of re-running the tests. By changing the commit message, the tests re-run, I think.

Yes, you are right.

**Review by abmerop — DISMISSED** [review]

**Comment by mattsinc** [comment]
@BobbyRBruce : any advice on how to dig further into this?  I see that:

Test: test-simpoints-se-restore-ALL-x86_64-opt Failed

However, I strongly suspect a commit only changing formatting and line spacing did not cause this test to fail.  But setting that aside, how do I run this test locally with the new setup to see what is happening and/or how do I get more details about what failed?  https://github.com/gem5/gem5/actions/runs/6787813182/job/18526299663 seems to just say it failed.

Seems like I need to run ./main.py run -vv -j$(nproc) gem5/gem5_library_example_tests with -vvv instead, but what is the correct place to run it from?

**Comment by mattsinc** [comment]
> @BobbyRBruce : any advice on how to dig further into this? I see that:
> 
> Test: test-simpoints-se-restore-ALL-x86_64-opt Failed
> 
> However, I strongly suspect a commit only changing formatting and line spacing did not cause this test to fail. But setting that aside, how do I run this test locally with the new setup to see what is happening and/or how do I get more details about what failed? https://github.com/gem5/gem5/actions/runs/6787813182/job/18526299663 seems to just say it failed.
> 
> Seems like I need to run ./main.py run -vv -j$(nproc) gem5/gem5_library_example_tests with -vvv instead, but what is the correct place to run it from?

Alternatively, it could just be the flakiness of the testers and I should rerun later?

**Comment by BobbyRBruce** [comment]
> > @BobbyRBruce : any advice on how to dig further into this? I see that:
> > Test: test-simpoints-se-restore-ALL-x86_64-opt Failed
> > However, I strongly suspect a commit only changing formatting and line spacing did not cause this test to fail. But setting that aside, how do I run this test locally with the new setup to see what is happening and/or how do I get more details about what failed? https://github.com/gem5/gem5/actions/runs/6787813182/job/18526299663 seems to just say it failed.
> > Seems like I need to run ./main.py run -vv -j$(nproc) gem5/gem5_library_example_tests with -vvv instead, but what is the correct place to run it from?
> 
> Alternatively, it could just be the flakiness of the testers and I should rerun later?

I don't know why this would fail. Re-running the tests.

**Comment by abmerop** [comment]
Is there a point of testing a change like this which just adds spaces?  Why not bypass the checks to save tester bandwidth?

**Comment by BobbyRBruce** [comment]
> Is there a point of testing a change like this which just adds spaces? Why not bypass the checks to save tester bandwidth?

There's no point in it. It's a waste of time. I just haven't implemented a way to skip CI tests for changes that don't contain actual code changes. Up until recently our bandwidth was ample (I hope it soon will be once we get Loupe back online), so it was never a priority.

**Comment by mattsinc** [comment]
> > Is there a point of testing a change like this which just adds spaces? Why not bypass the checks to save tester bandwidth?
> 
> There's no point in it. It's a waste of time. I just haven't implemented a way to skip CI tests for changes that don't contain actual code changes. Up until recently our bandwidth was ample (I hope it soon will be once we get Loupe back online), so it was never a priority.

Sorry @BobbyRBruce , but in the process of updating #540 over the weekend I found a few more formatting issues and pushed them here, now.  Not sure if there is a way to avoid running the tests again and again for formatting?

**Comment by BobbyRBruce** [comment]
> > > Is there a point of testing a change like this which just adds spaces? Why not bypass the checks to save tester bandwidth?
> > 
> > 
> > There's no point in it. It's a waste of time. I just haven't implemented a way to skip CI tests for changes that don't contain actual code changes. Up until recently our bandwidth was ample (I hope it soon will be once we get Loupe back online), so it was never a priority.
> 
> Sorry @BobbyRBruce , but in the process of updating #540 over the weekend I found a few more formatting issues and pushed them here, now. Not sure if there is a way to avoid running the tests again and again for formatting?

I think you can "force" a merge:

Screenshot 2023-11-13 at 9 35 17 AM

Click the "Merge without waiting for requirements" box and click "Merge pull request". I believe all maintainers have this power and it's ok to use it in cases where the outcome of the tests is moot.

**Comment by mattsinc** [comment]
> > > > Is there a point of testing a change like this which just adds spaces? Why not bypass the checks to save tester bandwidth?
> > > 
> > > 
> > > There's no point in it. It's a waste of time. I just haven't implemented a way to skip CI tests for changes that don't contain actual code changes. Up until recently our bandwidth was ample (I hope it soon will be once we get Loupe back online), so it was never a priority.
> > 
> > 
> > Sorry @BobbyRBruce , but in the process of updating #540 over the weekend I found a few more formatting issues and pushed them here, now. Not sure if there is a way to avoid running the tests again and again for formatting?
> 
> I think you can "force" a merge:
> Screenshot 2023-11-13 at 9 35 17 AM
> 
> Click the "Merge without waiting for requirements" box and click "Merge pull request". I believe all maintainers have this power and it's ok to use it in cases where the outcome of the tests is moot.

In my case the little checkbox line is missing.  I assume this is because it wants a rebase with develop?

**Comment by mattsinc** [comment]
> > > > > Is there a point of testing a change like this which just adds spaces? Why not bypass the checks to save tester bandwidth?
> > > > 
> > > > 
> > > > There's no point in it. It's a waste of time. I just haven't implemented a way to skip CI tests for changes that don't contain actual code changes. Up until recently our bandwidth was ample (I hope it soon will be once we get Loupe back online), so it was never a priority.
> > > 
> > > 
> > > Sorry @BobbyRBruce , but in the process of updating #540 over the weekend I found a few more formatting issues and pushed them here, now. Not sure if there is a way to avoid running the tests again and again for formatting?
> > 
> > 
> > I think you can "force" a merge:
> > Screenshot 2023-11-13 at 9 35 17 AM
> > Click the "Merge without waiting for requirements" box and click "Merge pull request". I believe all maintainers have this power and it's ok to use it in cases where the outcome of the tests is moot.
> 
> In my case the little checkbox line is missing. I assume this is because it wants a rebase with develop?

Nope, even now after rebasing it is still missing for me ...

**Comment by BobbyRBruce** [comment]
> > > > > > Is there a point of testing a change like this which just adds spaces? Why not bypass the checks to save tester bandwidth?
> > > > > 
> > > > > 
> > > > > There's no point in it. It's a waste of time. I just haven't implemented a way to skip CI tests for changes that don't contain actual code changes. Up until recently our bandwidth was ample (I hope it soon will be once we get Loupe back online), so it was never a priority.
> > > > 
> > > > 
> > > > Sorry @BobbyRBruce , but in the process of updating #540 over the weekend I found a few more formatting issues and pushed them here, now. Not sure if there is a way to avoid running the tests again and again for formatting?
> > > 
> > > 
> > > I think you can "force" a merge:
> > > Screenshot 2023-11-13 at 9 35 17 AM
> > > Click the "Merge without waiting for requirements" box and click "Merge pull request". I believe all maintainers have this power and it's ok to use it in cases where the outcome of the tests is moot.
> > 
> > 
> > In my case the little checkbox line is missing. I assume this is because it wants a rebase with develop?
> 
> Nope, even now after rebasing it is still missing for me ...

My mistake, i just checked the permissions. You're a "Maintainer" but you need to be an "Administrator" or "Owner" to see that box, and there's nothing I can do about that as allowing "Maintainer" level users this privilege is only available on GitHub Enterprise.

**Comment by mattsinc** [comment]
> > > > > > > Is there a point of testing a change like this which just adds spaces? Why not bypass the checks to save tester bandwidth?
> > > > > > 
> > > > > > 
> > > > > > There's no point in it. It's a waste of time. I just haven't implemented a way to skip CI tests for changes that don't contain actual code changes. Up until recently our bandwidth was ample (I hope it soon will be once we get Loupe back online), so it was never a priority.
> > > > > 
> > > > > 
> > > > > Sorry @BobbyRBruce , but in the process of updating #540 over the weekend I found a few more formatting issues and pushed them here, now. Not sure if there is a way to avoid running the tests again and again for formatting?
> > > > 
> > > > 
> > > > I think you can "force" a merge:
> > > > Screenshot 2023-11-13 at 9 35 17 AM
> > > > Click the "Merge without waiting for requirements" box and click "Merge pull request". I believe all maintainers have this power and it's ok to use it in cases where the outcome of the tests is moot.
> > > 
> > > 
> > > In my case the little checkbox line is missing. I assume this is because it wants a rebase with develop?
> > 
> > 
> > Nope, even now after rebasing it is still missing for me ...
> 
> My mistake, i just checked the permissions. You're a "Maintainer" but you need to be an "Administrator" or "Owner" to see that box, and there's nothing I can do about that as allowing "Maintainer" level users this privilege is only available on GitHub Enterprise.

No worries.  Feel free to merge this yourself though if it will save bandwidth.

**Review by abmerop — APPROVED** [review]

**Comment by mattsinc** [comment]
Rebased to head of develop again.  @BobbyRBruce this is another formatting one, if tester bandwidth is still crunched.

**Pull request merged**
